orchestrator/db/filters/process.py

```diff
@@ -128,32 +130,31 @@ def workflow_name_filter(query: QueryType, value: str) -> QueryType:
         "istask": generic_bool_filter(ProcessTable.is_task),  # TODO: will be removed in 1.4
         "status": generic_values_in_column_filter(ProcessTable.last_status),  # TODO: will be removed in 1.4
         "last_status": generic_values_in_column_filter(ProcessTable.last_status),  # TODO: will be removed in 1.4
-        "tag": tag_filter,  # TODO: will be removed in 1.4
```

Comment:
I forgot why this was marked as `TODO: will be removed in 1.4`. @tjeerddie do you remember?